### PR TITLE
[Demo] Reset transform device when changing video input quality

### PR DESCRIPTION
**Pull request by ltrung**
**Issue:**
In the demo preview page, applying a video filter then changing the video input quality will cause the video preview to become blank.

**Description of changes:**
Make sure to stop the transform device when changing video input quality.

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
- Go to the meeting video preview.
- Apply a filter
- Changing video input quality 
- Verify that the video is still shown and the filter is applied.

**Checklist:**

1. Have you successfully run `npm run build:release` locally? Yes.

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 1]
```diff
@@ -986,6 +986,10 @@ export class DemoMeetingApp
           break;
       }
       try {
+        if (this.chosenVideoTransformDevice) {
+          await this.chosenVideoTransformDevice.stop();
```
[the remark this answers is not in the document]
Comment (hensmi-amazon) [reply demos/browser/app/meetingV2/meetingV2.ts:990]: Sgtm, maybe is something we can fix for v3?  I feel like a video transform pipeline should be something that can handle changes in resolution.

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 2]
```diff
@@ -986,6 +986,10 @@ export class DemoMeetingApp
...
+          this.chosenVideoTransformDevice = null;
+        }
         await this.openVideoInputFromSelection(videoInput.value, true);
```
Comment (nainkunal933) [inline demos/browser/app/meetingV2/meetingV2.ts:993]: After this call to `openVideoInputFromSelection` we should add a check for the selected video filter and create the transform device. My concern is if we don't do that then the UI and the video stream might not be in sync.

File: guides/10_Video_Processor.md [hunk 3]
```diff
@@ -59,6 +59,8 @@ After stopping the video processing, the inner `Device` will be released by devi
 
 After `DefaultVideoTransformDevice` is no longer used by device controller, call `DefaultVideoTransformDevice.stop` to release the `VideoProcessor`s and underlying pipeline. After `stop` is called, users must discard the `DefaultVideoTransformDevice`.`DefaultVideoTransformDevice.stop` is necessary to release the internal resources.
 
+Users would also need to call `DefaultVideoTransformDevice.stop` and construct a new `DefaultVideoTransformDevice` when they want to change video processors or change the video input quality.
```
Comment (nainkunal933) [inline guides/10_Video_Processor.md:62]: Thank you for adding documentation :)

File: demos/browser/app/meetingV2/meetingV2.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ltrung) [reply demos/browser/app/meetingV2/meetingV2.ts:990]: Will create a SIM to track this.

**Pull request merged**